apps/api/src/http/controllers/food-search.controller.ts

```diff
@@ -62,6 +77,26 @@ const foodSearchController = ({
     };
   }
 
+  // API endpoint for rebuilding the food index
+  const rebuildFoodIndex = async (req: Request, res: Response): Promise<void> => {
+    logger.debug(`\n\n\nRebuilding ${JSON.stringify(req.body)} food index\n\n\n`);
```

Comment:
Could be remove these additional lines in log msgs, please? Makes the log file non-standard for machine reading.